spec/cassettes/github_dashboard.yml

```diff
@@ -2,26 +2,26 @@
 http_interactions:
 - request:
     method: get
-    uri: https://api.github.com/user/repos
+    uri: https://api.github.com/user/repos?sort=updated
     body:
       encoding: US-ASCII
       string: ''
     headers:
       Authorization:
       - token <GITHUB-TOKEN>
+      Accept:
```

Comment:
Subtle change, but important to not introduce an unpleasant surprise should github change their default api version.